### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by zieeco**
# Portfolio: desktop version 
**In this milestone, I;**
- Added some content and changed the footer to a section  and nested it within the main tag in the index.html file
- Added the styling using media queries, and also add a breakpoint for Ipad in the main.css file
- Also, ensured there were no linters error

This is the desktop version of my portfolio project built with responsiveness in mind.

### CODE REVIEW

File: CSS/main.css [hunk 1]
```diff
@@ -433,3 +440,242 @@ main {
...
+    max-width: 477px;
+  }
+
+  .contact-form {
+    max-width: 350px;
+  }
+}
+
+/* BIG SCREENS */
+@media screen  and (min-width: 991px) {
```
Comment (AshakaE) [inline CSS/main.css]: - [ ] Please pick a value from `992 - 1100px` , the present value is too low 🙏

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @zieeco  🙋‍♀️, 

Good job so far! 👌
Great job on the design 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

#  Required Changes ♻️ 🚧
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: CSS/main.css (hunk 1 above)
Comment (zieeco) [reply CSS/main.css]: Thanks for your review. I have increased the value to 992px as you suggested

**Review by mke2111 — APPROVED** [review]
### Hi @zieeco,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**